packages/web/src/models/test.ts

```diff
@@ -65,3 +67,12 @@ export type TestSuiteStepTest = {
   namespace: Test['namespace'];
   type?: Test['type'];
 };
+
+export enum TestsView {
+  LIST = 'list',
+  TABLE = 'table',
```

Comment:
```suggestion
  List = 'list',
  Table = 'table',
```

Reply:
Removed it completely as not needed anymore